cnv/cnv_users_guide/cnv-edit-vms.adoc

```diff
@@ -11,5 +11,4 @@ a subset of the parameters in the *Virtual Machine Overview* of the web console.
 include::modules/cnv-editing-vm-web.adoc[leveloffset=+1]
 include::modules/cnv-editing-vm-yaml-web.adoc[leveloffset=+1]
 include::modules/cnv-editing-vm-cli.adoc[leveloffset=+1]
-
-include::modules/cnv-add-disk-to-vm.adoc[leveloffset=+1]
```

Comment:
@bgaydosrh I might be missing something, but did you mean to remove the add-disk module in these two assemblies?

Reply:
Yes, this PR should only have NIC but eventually will contain disk, NIC, and CDROM once all PR have been merged.